dist/profile/templates/confluence/vhost.conf

```
@@ -2849,12 +2849,18 @@ RewriteRule "^/display/JENKINS/ZMQ\+Event\+Publisher\+Plugin$" "https://plugins.
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Zoom\+Plugin$" "https://plugins.jenkins.io/zoom" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
-RewriteRule "^/display/JENKINS/IBM\+zOS\+Connector$" "https://plugins.jenkins.io/zos-connector" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/IBM\+zOS\+Connector\+Plugin$" "https://plugins.jenkins.io/zos-connector" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/violation\+comments\+to\+gitlab\+Plugin$" "https://plugins.jenkins.io/violation-comments-to-gitlab" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/claim\+Plugin$" "https://plugins.jenkins.io/claim/" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/appcenter\+Plugin$" "https://plugins.jenkins.io/appcenter/" [NC,L,QSA,R=301]
```

The page https://wiki.jenkins.io/display/JENKINS/appcenter+Plugin does not exist on the wiki.

The page https://wiki.jenkins.io/display/JENKINS/App+Center+Plugin does exist and should be the source of this redirect

```suggestion
RewriteRule "^/display/JENKINS/App\+Center\+Plugin$" "https://plugins.jenkins.io/appcenter/" [NC,L,QSA,R=301]
```

---

So I got the information in https://github.com/jenkins-infra/jenkins.io/issues/3773 wrong.

* Page to Migrate: https://wiki.jenkins.io/display/JENKINS/App+Center+Plugin
* Redirect to the destination: https://plugins.jenkins.io/appcenter/

I thought that **Redirect to** was the link that we should add to the `vhost.conf` That's the reason for the mistake on the other two links. I'm accepting your commit suggestion. Thank you.

---

**Update** - I can't accept your commit because the PR is merge. I'll create a new PR to fix this then.

---

Fixed by #1591